pydantic_ai_slim/pydantic_ai/models/openai.py

```diff
@@ -2240,6 +2295,34 @@ def _map_web_search_tool_call(
     )
 
 
+def _map_file_search_tool_call(
+    item: responses.ResponseFileSearchToolCall,
+    provider_name: str,
+) -> tuple[BuiltinToolCallPart, BuiltinToolReturnPart]:
+    args = {'queries': item.queries}
+
+    result: dict[str, Any] = {
+        'status': item.status,
+    }
+    if item.results is not None:
```

Comment:
How do we get results to be included? Should we have a model setting like the existing `openai_include_web_search_sources`? (Search the code and docs for examples of how it works, and where we would need to mention the new `openai_include_file_search_results` as well)